layers/+lang/typescript/README.org

```diff
@@ -170,24 +170,25 @@ then set the variable =typescript-lsp-linter= to =nil=.
 
 | Key binding | Description                                                  |
 |-------------+--------------------------------------------------------------|
-| ~SPC m =~   | reformat the buffer                                          |
-| ~SPC m E d~ | add =tslint:disable-next-line= at point                      |
-| ~SPC m E e~ | fix thing at point                                           |
-| ~SPC m g b~ | jump back                                                    |
-| ~SPC m g g~ | jump to entity's definition                                  |
-| ~SPC m g t~ | jump to entity's type definition                             |
-| ~SPC m g u~ | references                                                   |
-| ~SPC m h h~ | documentation at point                                       |
-| ~SPC m r i~ | organize imports                                             |
-| ~SPC m r r~ | rename symbol                                                |
-| ~SPC m r f~ | rename file                                                  |
-| ~SPC m s p~ | send selected region or current buffer to the web playground |
-| ~SPC m s r~ | restart server                                               |
+| ~SPC m = t~   | reformat the buffer                                          |
```

Comment:
I think this binding is wrong, it should rather be `SPC m =` independent of who does the formating.